tests/basic-test.sh

```diff
@@ -793,7 +793,8 @@ rm files -rf && mkdir files
 mkdir files/worldwritable-dir
 chmod a+w files/worldwritable-dir
 $OSTREE commit ${COMMIT_ARGS} -b content-with-dir-world-writable --tree=dir=files
-$OSTREE fsck
+# FIXME(lucab): this seems to fail in unprivileged bare mode.
+#$OSTREE fsck
```

Comment:
Let's conditionalize this on `if is_bare_user_only_repo` like below?

Reply:
I thing this does work in bare-user-only mode. That's the reason why I did not see this failing when I added it initially.
I can make this conditional on `have_selinux_relabel` and keep the FIXME note, if you prefer.

Reply:
Hmm you're saying the fsck only fails if selinux/xattrs?   Sure, we can make that conditional but...oh I see, there's also e.g.
`echo "ok # SKIP checksums won't match up in bare-user-only"` above here.

I think we should fix things so that `fsck` *does* work in that case too.  But it's fine to make it conditional too for now and then we circle back and fix it and that other conditional too.

Reply:
Ack, I made this conditional and filed https://github.com/ostreedev/ostree/issues/2432 for later.